curated_transformers/models/transformer_model.py

```diff
@@ -70,6 +70,52 @@ def build_albert_transformer_model_v1(
     mixed_precision: bool = False,
     grad_scaler_config: dict = SimpleFrozenDict(),
 ) -> TransformerModelT:
+    """Construct an ALBERT transformer model.
+
+    This model must be separately initialized using
```

Comment:
I think this is worded too strongly. It is not necessary and in some cases desirable to not use a loader (e.g. train a model from scratch/distillation).

Reply:
I've removed that sentence.